app/helpers/works_helper.rb

```diff
@@ -191,4 +191,13 @@ def chapter_total_display_with_link(work)
       chapter_total_display(work)
     end
   end
+
+  def get_open_assignments(user)
+    if user == User.orphan_account
+      # orphan_account has too many pseuds to feasibly get. This lets admins access the work form without timing out.
+      return user.offer_assignments.undefaulted.unstarted.sent + user.pinch_hit_assignments.undefaulted.unstarted.sent
```

Comment:
Not super on point with how assignments work, so might be asking the obvious here but... How can the orphan account have open assignments? Should it? What would happen if we were to just return an empty list there?

Reply:
In theory, someone could assign orphan_account a pinch hit in an exchange.

In reality, though, I checked staging and production, and the only assignment orphan_account had was the one I just gave it to test whether it could be assigned a pinch hit:

```
user = User.orphan_account
 => #<User id:...
user.offer_assignments.undefaulted.unstarted.sent.size
 => 0 
user.offer_assignments.size
 => 0 
user.pinch_hit_assignments.undefaulted.unstarted.sent.size
 => 0 

user.pinch_hit_assignments.undefaulted.unstarted.sent.size
 => 1 
```

Reply:
I am wondering then whether ceithir's empty list could work.

Reply:
I think it would be fine to do it that way, yeah.